cmd/yurthub/app/config/config.go

```diff
@@ -200,7 +203,20 @@ func createSharedInformers(proxyAddr string) (informers.SharedInformerFactory, y
 		return nil, nil, err
 	}
 
-	return informers.NewSharedInformerFactory(client, 24*time.Hour), yurtinformers.NewSharedInformerFactory(yurtClient, 24*time.Hour), nil
+	if len(nodePoolName) == 0 {
+		return informers.NewSharedInformerFactory(client, 24*time.Hour), yurtinformers.NewSharedInformerFactory(yurtClient, 24*time.Hour), nil
+	}
+	labelSelector := labels.NewSelector()
+	nodePoolLabel, err := labels.NewRequirement("metadata.name", selection.Equals, []string{nodePoolName})
+	if err != nil {
+		return nil, nil, err
+	}
+	labelSelector = labelSelector.Add(*nodePoolLabel)
+	yurtSharedInformerFactory := yurtinformers.NewSharedInformerFactoryWithOptions(yurtClient, 24*time.Hour,
+		yurtinformers.WithTweakListOptions(func(options *metav1.ListOptions) {
+			options.LabelSelector = labelSelector.String()
```

Comment:
please use `FieldSelector` instead `LabelSelector` for `metadata.name`,  so the code like as following:
```
options.FieldSelector = fields.Set{"metadata.name": string(nodePoolName)}.String()
```

Reply:
Thanks for your help. I have already modified.